client/src/pages/profile/index.tsx

```diff
@@ -0,0 +1,170 @@
+import ApplicationLayout from 'layouts/application';
+import Head from 'next/head';
+import { Label, Input } from 'components/forms';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as yup from 'yup';
+import { useForm } from 'react-hook-form';
+import { useMutation } from 'react-query';
+import { useCallback } from 'react';
+import { apiService } from 'services/api';
+
+type UserProfilePayload = {
+  fname?: string;
+  lname?: string;
+  email?: string;
+  password?: string;
+};
```

Comment:
this is maybe already defined in other files. Please, ensure is not repeated and in case it is, move it to a `types.d.ts` file